examples/camera-controller/device-manager/AVStreamManagement.cpp

```diff
@@ -0,0 +1,186 @@
+/*
+ *   Copyright (c) 2025 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+
+#include "AVStreamManagement.h"
+#include "DeviceManager.h"
+
+using namespace ::chip;
+
+namespace {
+
+constexpr uint16_t kMinFrameRate   = 30;
```

Comment:
Instead of hardcoding, it might be better to read the relevant attributes to set the values for the Allocate command params?

Reply:
This needs a new flow, track this as a follow-up with https://github.com/project-chip/connectedhomeip/issues/39233

Reply:
A follow-up is fine. Basically collect the supported params and ranges from the camera attributes before populating the Allocate params.